.github/workflows/authors.yml

```diff
@@ -6,6 +6,9 @@ on:
 
   workflow_dispatch:
 
+permissions:
+  contents: read
```

Comment:
This workflow might open a pull request. Is this setting correct?

Reply:
The setting is correct because the credential used to open the PR is not the `GITHUB_TOKEN`. It is a different secret which has been explicitly setup called `GH_USER_TOKEN`. 

https://github.com/nodejs/node/blob/19cc1e891e2a03e94c642b375c213143daf69beb/.github/workflows/authors.yml#L26